editor/src/components/inspector/common/longhand-shorthand-hooks.ts

```diff
@@ -245,14 +231,92 @@ export function useInspectorInfoLonghandShorthand<
       controlStyles: getControlStyles(controlStatus),
       onSubmitValue: onSubmitValue,
       onTransientSubmitValue: onTransientSubmitValue,
-      onUnsetValues: onUnsetValues,
+      onUnsetValues: NO_OP,
       orderedPropKeys: orderedPropKeys,
     }
   })
 
-  return mapArrayToDictionary(
+  const longhandResultsDictionary = mapArrayToDictionary(
     longhandResults,
     (_, index) => longhands[index],
     (result) => result,
   )
+
+  const longhandResultsWithUnset = objectMap((longhandResult, longhandToUnset) => {
+    const onUnsetValues = () => {
+      const allPropKeysEqual = allOrderedPropKeys.every((propKeys) => {
+        return arrayEquals(propKeys, allOrderedPropKeys[0])
+      })
+      if (!allPropKeysEqual) {
+        // we do nothing for now. we cannot ensure that we can make a sensible update and surface it on the UI as well
```

Comment:
Does this mean a silent failure?

Reply:
yes. this is code I copy pasted from here: https://github.com/concrete-utopia/utopia/blob/7a97a87663b77a7573614a3b42b0af5081913400/editor/src/components/inspector/common/longhand-shorthand-hooks.ts#L166

we agreed to not solve the not-identical multiselect problem for now

Reply:
more info in the **Note:** section of this PR https://github.com/concrete-utopia/utopia/pull/925 and here https://github.com/concrete-utopia/utopia/pull/925#issuecomment-781590033